data.json

```diff
@@ -471,11 +471,11 @@
       "navbar-with-sidebar": {
         "navbar": {
           "logo": {
-            "url": "../img/thescenic/header-logo.png",
+            "url": "../img/components/sidebar-example-logo.png",
```

Comment:
you will need to updated these in the appropriate files in the `hl-partials` folder